javascript/chronometer.js

```diff
@@ -1,35 +1,83 @@
 class Chronometer {
   constructor() {
     // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
+
+    this.currentTimeMili = 0;
+    this.intervalIdMili = null;
   }
 
   start(callback) {
     // ... your code goes here
+    const incrementOne = callback => {
+      this.currentTime++;
+    }
+    this.intervalId = setInterval(incrementOne, 1000)
+  }
+
+  // ===> BONUS
+  startMili () {
+    const incrementOneMili = callback => {
+      this.currentTimeMili++;
+    }
+    this.intervalIdMili = setInterval(incrementOneMili, 10)
+  }
+
+  getMili () {
+    let milis = this.currentTimeMili % 100;
+    return milis;
   }
 
+  splitMili () {
+    let value1 = this.computeTwoDigitNumber(this.getMili());
+    return `${value1}`
```

Comment:
No necesitamos backticks aqui para hacer uso de javascript por que ya estamos en un archivo .js
```suggestion 
    return value1
```